.github/workflows/unit_testing.yml

```diff
@@ -1,20 +1,34 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
 name: Unit Test Workflow
+
 on:
   push:
-    branches:
-      - dev
-      - main
+    branches: [ "dev" ]
+  pull_request:
+    branches: [ "dev" ]
+
+env:
+    WORKING_DIRECTORY: ./
```

Comment:
This env would become unnecessary, as I commented in https://github.com/ntohq/buefy-next/pull/137/files#r1389008050.